docs/contributing.md

```diff
@@ -4,6 +4,29 @@ Contributions are welcome, assuming that they align with the general design goal
 
 Unless you just want to contribute a small bug fix, it is a good idea to start by opening an issue and discuss your idea with the maintainers. This maximizes the chances that your contribution will be accepted once you open a pull request.
 
+## Community Governance
+
+vis.gl is part of the [Urban Computing Foundation](https://uc.foundation/). See the organization's [Technical Charter](https://github.com/visgl/tsc/blob/master/Technical%20Charter.md).
+
+### Technical Steering Committee
+
+loaders.gl development is governed by the vis.gl Technical Steering Committee (TSC). Current members:
+
+- [Ib Green](https://github.com/ibgreen)
```

Comment:
I don't think we should duplicate the member list of the TSC here, that sounds like a maintenance trap. Rather link to that doc in the tsc repo. 

And focus on listing the key maintainers for this repo. We can move these names to the next section.

Reply:
Ok. The TSC member list is still in the deck.gl `Contributing.md`